External/Plugins/ASCompletion/Model/ASFileParser.cs

```diff
@@ -1112,6 +1112,16 @@ public void ParseSrc(FileModel fileModel, string ba, bool allowBaReExtract)
                     else if (inType)
                     {
                         foundColon = false;
+                        if (haXe && c1 == '>' && ba[i - 2] == '-' && inAnonType)
```

Comment:
Since the first condition is haxe, it looks like this if can be moved some lines down.